onnxruntime/core/framework/sequential_execution_plan.h

```diff
@@ -36,11 +36,17 @@ struct AllocPlanPerValue {
   // if the value is used in async kernel, a fence object would be created
   // note the fence object would be shared between MLValues reusing the same buffer
   bool create_fence_if_async{false};
-#if !defined(ORT_MINIMAL_BUILD) && defined(ORT_MEMORY_PROFILE) 
+#if !defined(ORT_MINIMAL_BUILD) && defined(ORT_MEMORY_PROFILE)
   IntervalT life_interval{0, 0};
   IntervalT allocate_interval{0, 0};
   OrtValueIndex inplace_reuse{-1}; //No in-place reuse
 #endif
+#ifdef ENABLE_TRAINING
+  // is_strided_tensor indicates if this OrtValue is strided tensor.
+  // If alloc_kind is kResue, it reuses one of the node inputs (like Expand),
+  // if alloc_kind is kAllocate, it will only allocate required buffer size (like ConstantOfShape).
```

Comment:
>// if alloc_kind is kAllocate, it will only allocate required buffer size (like ConstantOfShape). [](http://example.com/codeflow?start=2&length=97)

I only see the kReuse branch,this branch is not supported yet, right?